docs/content/en/docs/fleet-manager/backup/backup.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Unified Backup"
+linkTitle: "Unified Backup"
+weight: 20
+description: >
+  A comprehensive guide on Kurator's Unified Backup solution, providing an overview and practical implementation steps.
+---
+
+Unified Backup offers a concise, one-click solution to back up application and cluster resources across all clusters in Fleet,
+supporting varying granularities from individual namespaces to multiple namespaces, and up to entire clusters.
+This ensures consistent data protection and facilitates swift recovery in a distributed cloud-native infrastructure.
+
+## Introduction
+
+### Backup Types
+
+Unified Backup supports two primary backup methods to cater to different requirements, ensuring data is safely stored and can be restored when necessary.
+
+#### Immediate Backup
+
+- **Use Case**: Essential for sudden changes, like after a significant data update or before implementing system alterations.
+- **Functionality**: Triggers a one-time backup on demand.
+
+#### Scheduled Backup
+
+- **Use Case**: Regularly back up dynamic data to safeguard against accidental losses, maintain compliance, and enable efficient disaster recovery.
+- **Functionality**: Set automated backups at specific intervals using cron expressions.
+
+### Advanced Backup Options
+
+- **Specific Cluster Backup within Fleet**: Users can choose one or more particular clusters within the Fleet for backup.
+- **Resource Filtering**: Kurator offers filtering options for more precise backups, allowing users to define criteria based on attributes like name, namespace, or label.
+
+> **Note**: Kurator supports backing up and restoring Kubernetes volumes attached to pods directly from the file system. Snapshot support is currently unavailable. Refer to the documentation [FSB](https://velero.io/docs/v1.11/file-system-backup/) for more information.
+
+## How to Perform a Unified Backup
+
+
+In the following sections, we will present a hands-on demonstration. 
+Before proceeding with the content, ensure you have successfully installed the backup plugin as per the [backup plugin installation guide](/docs/fleet-manager/backup/backup-plugin).
+
+
+For our demonstration, we'll utilize the fleet and two kind clusters which created in [backup plugin installation guide](/docs/fleet-manager/backup/backup-plugin). 
+
+Let's start by deploying a busybox example:
+
+```console
+kubectl apply -f examples/backup/busybox.yaml --kubeconfig=/root/.kube/kurator-member1.config
+kubectl apply -f examples/backup/busybox.yaml --kubeconfig=/root/.kube/kurator-member2.config
```

Comment:
can you use kurator mechanism, application way

Reply:
The current document is focused on backups. I'm not sure if it's appropriate to include other components here, especially since we might make adjustments to the application content in the CI/CD area.

Reply:
Kurator, we provide an entry for multi clusters. If users can operate each cluster directly, why would they even make trouble to try kurator?

https://kurator.dev/docs/fleet-manager/policy/